spec/controllers/password_reset_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe PasswordResetsController do
+  describe 'GET show' do
+    it 'renders show template if token is valid' do
```

Comment:
This should be standard behaviour of Rails to render show template when you are in show action. I recommend to not test it.